db/migrate/20230310014206_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :postId
+      
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] Kindly ensure that you remove this `postId` column from the `likes` table. According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) the `post_id` should a `foreign_key` as you did in the `db/migrate/20230310022116_add_post_ref_to_likes.rb` migration. 
That means the only issue is to remove this `postId` non-existed column from the  [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) in your `likes` table, then you can update your schema again. **An example on how to proceed just refer this [article](https://stackoverflow.com/questions/2831059/how-to-drop-columns-using-rails-migration)**